api/pom.xml

```diff
@@ -147,25 +166,36 @@
                         <artifactId>maven-compiler-plugin</artifactId>
                         <executions>
                             <execution>
-                                <id>jdk-9</id>
+                                <id>jdk-11</id>
                                 <goals>
                                     <goal>compile</goal>
                                 </goals>
                                 <configuration>
-                                    <source>9</source>
-                                    <target>9</target>
+                                    <source>11</source>
+                                    <target>11</target>
                                     <includes>
                                         <include>module-info.java</include>
                                     </includes>
                                 </configuration>
                             </execution>
                         </executions>
                     </plugin>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-javadoc-plugin</artifactId>
+                        <version>3.1.0</version>
+                        <configuration>
+                            <detectJavaApiLink>false</detectJavaApiLink>
+                            <links>
+                                <link>https://docs.oracle.com/en/java/javase/11/docs/api</link>
+                            </links>
+                        </configuration>
+                    </plugin>
                 </plugins>
             </build>
         </profile>
         <profile>
-            <id>non-jdk9</id>
+            <id>jdk8</id>
             <activation>
                 <jdk>(,9)</jdk>
```

Comment:
if we remove java 9 profile we should update this to `(,11)`

Reply:
No. This profile works only with java8. It's why I added separate java9 profile only to keep a compatibility.